Sample Application/SampleApplication/Models/NetworkEndpoint.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  NetworkEndpoint.swift
+//  SampleApplication
+//
+//  Created by Alex Ioja-Yang on 24/10/2022.
+//
+
+import Foundation
+import CheckoutNetwork
+
+enum NetworkEndpoint: NetworkPath {
+
+    case authentication
+
+    func url() -> URL {
+        switch self {
+        case .authentication:
+            return URL(string: "https://api.sandbox.checkout.com/issuing/access/connect/token")!
```

Comment:
This is actually hardcoded to the sandbox. Whilst I think we are also supporting Prod (with testers own key & secret), they couldn't get that far if we don't make this URL return based on environment.

Reply:
Also this must be used in sample app to create tokens and show a end to end flow, which ideally merchants will do this from their server.

Reply:
done @alex-ioja-yang-cko 

@deepesh-vasthimal-cko that footnote had been appended in `Authenticator.swift` file and also I now added a reference to that in this file, too.